### DISCUSSION TITLE
Prose not working in Safari

### DISCUSSION THREAD

**Discussion by Applicafroguy** [tailwindlabs/tailwindcss#9090] [Help] [answered]
**Safari** : Version 13.1.3 (15609.4.1)

Typography plugins doesn't seem to work in in Safari, on Chrome and Firefox it works correct

**Comment by wongjn** [comment, marked as answer]
Unfortunately yes. `@tailwindcss/typography` uses the [`:where()` selector](https://developer.mozilla.org/en-US/docs/Web/CSS/:where) heavily and the earliest version of Safari and Safari iOS to support this seems to be 14.

**Comment by kylef-fotf** [comment]
I would like to resurface this question to ask: How do you create a (minimal) working fallback for this situation?

How should someone set a fallback, so certain elements like headings and paragraphs, get their sizing and color on devices that don't support `:where()`?
